### DISCUSSION TITLE
KI-Strategie Senderkauf

### DISCUSSION THREAD

**Discussion by nittka** [TVTower/TVTower#598] [Ideas]
(Auch) Die Testspiele zeigen, dass der Senderausbau einen entscheidenden Einfluss auf die Performance der KI hat - höhere Werbeeinnahmen + exclusive Zuschauer für schnelleren Image-Anstieg. Allerdings hat die aktuelle randomisierte Standortsuche einige entscheidende, langfristige Nachteile. Zunächst ist ja viel Platz und die ersten Antennen werden schön verstreut und ohne Überlappung verteilt. Aber bevor genug Geld für die nächste Sendegenehmigung da ist, gibt es im Startbundesland schon viele kleine Lücken und viele Antennen mit Überlappung (weil die eben doch "attraktiver" waren).
Ein nachträgliches Optimieren der Senderkarte ist dann auch nicht einfach (und ggf. sehr teuer). Man könnte zwar Sender mit geringer exklusiver Zuschauerzahl wieder verkaufen/entsorgen, die dadurch entstehenden Lücken lassen sich aber auch wieder schlecht ohne Überlappung schließen.
Hinzu kommt, dass die "permanente" Analyse sämtlicher Spielerantennen und die Prüfung von randomisierten Antennenpositionen sehr zeitaufwändig ist.

Wenn die KI schon am Anfang die Karte "aufteilen" würde - wo sollen irgendwann mal meine Antennen hin - und dann immer von dieser Auswahl die gerade attraktivste baut, wäre viele dieser Probleme keine mehr. Die Senderkarte sieht dann vielleicht nicht mehr interessant aus, weil alle KIs dieselben Standorte wählen, aber selbst damit ist nicht gesagt, dass die Antennen in derselben Reihenfolge gebaut werden (Budget).
Und auch die Budgetplanung und Investitionsstrategie wäre leichter zu bearbeiten, weil man sich eben nicht im vermeintlich attraktiven Überlappungsausbauten rumschlagen muss.

Ich werde diese Idee wahrscheinlich ohnehin mal umsetzen, wollte aber schonmal Feedback einholen.

**Comment by GWRon** [comment]
Naja ein "natuerliches" Ausbreiten wuerde man ueber die Erhoehung des "Radiuses" von der Erstantenne erreichen ...

Statt der Erstantenne kann man natuerlich auch andere "beacons" raussuchen und sich davon ausgehend dann erweitern.

**Comment by nittka** [comment]
Eine erste Variante ist implementiert. Es wird eine Abdeckung des Sendegebiets mit Antennen vorausberechnet. Damit nicht alle dieselben Positionen haben, in unterschiedlich geneigten "Reihen" gebaut. Um die Lücken zwischen 3 in einem gleichseitigen Dreick angeordneten Antennen zu minimieren, wird eine gewisse Überlappung in Kauf genommen.
